.github/workflows/release.yaml

```diff
@@ -44,31 +58,24 @@ jobs:
         run: pytest tests/tests_unit -n8 --dist loadscope --maxfail 10 -m 'not dsl'
           --test-deps-only-core
 
-  test_full_and_build_and_release:
-    runs-on: ubuntu-latest
+  test_full:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest]
```

Comment:
**[Comment not about this PR]** Since we require PRs to be up-to-date (with `master`) before merging, and `test_full` runs on PRs, it does not make sense to me to re-run the full test suite on merges to `master` / releases. It makes chain releases/merges twice as slow to complete and just more error-prone imo without being more secure. Wdyt @erlendvollset ?